packages/sanity/src/core/form/types/asserters.ts

```diff
@@ -17,32 +17,39 @@ import {
 } from './inputProps'
 import {ItemProps, ObjectItemProps} from './itemProps'
 
+/** @beta */
```

Comment:
I like that these are exported, they are useful and will reduce boilerplate when used in plugins an the like 👍